Header header
uint16 x
uint16 y
uint16 nx
uint16 ny

